### PR TITLE
Data access technical debt

### LINKED ISSUE #518: Better for you to follow the stile guide: DataAccess
## Description:
For the long time we didn't following the style, namely chapter _Names_ rule 2. It's time to change that!
This should remove such warnings:
```log
/home/runner/work/Juniorgram/Juniorgram/Network/Public/Include/Network/Primitives.hpp: In constructor ‘Network::ChannelInfo::ChannelInfo(uint64_t, uint64_t, const string&)’:
/home/runner/work/Juniorgram/Juniorgram/Network/Public/Include/Network/Primitives.hpp:32:9: warning: declaration of ‘channelName’ shadows a member of ‘Network::ChannelInfo’ [-Wshadow]
   32 |         : creatorID(creatorID), channelID(channelID), channelName(channelName)
      |     
```

## Acceptance Criteria:
Follow the rule and rename variables in proper way with underscore.

### PR DESCRIPTION
**Pull request by ivanboika**
Task: For the long time we didn't following the style, namely chapter Names rule 2. It's time to change that!
Acceptance Criteria:
Follow the rule and rename variables in proper way with underscore.

### CODE REVIEW

**Review by memorisecodead — APPROVED** [review]
Somethings good

File: DataAccess/DataAccess.SQLite/LiteAdapter.cpp [hunk 1]
```diff
@@ -7,29 +7,29 @@ namespace DataAccess
 {
 LiteAdapter::LiteAdapter(const std::string_view& db_name)
 {
-    db_connection  = std::make_unique<QSqlDatabase>();
-    *db_connection = QSqlDatabase::addDatabase("QSQLITE");
-    db_connection->setDatabaseName(QString::fromStdString(std::string(db_name)));
-    db_connection->open();
+    _DBConnection  = std::make_unique<QSqlDatabase>();
+    *_DBConnection = QSqlDatabase::addDatabase("QSQLITE");
```
Comment (Orange-hanter) [inline DataAccess/DataAccess.SQLite/LiteAdapter.cpp]: Camel case violation

File: DataAccess/DataAccess.SQLite/LiteAdapter.cpp [hunk 2]
```diff
@@ -7,29 +7,29 @@ namespace DataAccess
...
+    _DBConnection->setDatabaseName(QString::fromStdString(std::string(db_name)));
+    _DBConnection->open();
 }
 
 std::shared_ptr<LiteAdapter> LiteAdapter::Instance(const std::string_view& db_name)
 {
-    std::scoped_lock<std::mutex> lock(mtx);
-    if (sqlite_instance == nullptr)
+    std::scoped_lock<std::mutex> lock(_mtx);
+    if (_SQLiteInstance == nullptr)
```
Comment (Orange-hanter) [inline DataAccess/DataAccess.SQLite/LiteAdapter.cpp]: Camel case violation

File: DataAccess/Public/Include/DataAccess/SQLStatements.hpp [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
-#include <Utility/SQLUtility.hpp>
-#include <FileLogger.hpp>
 #include <any>
 #include <iostream>
 #include <optional>
 #include <sstream>
 
+#include <Utility/Exception.hpp>
+#include <Utility/SQLUtility.hpp>
+
```
Comment (Orange-hanter) [inline DataAccess/Public/Include/DataAccess/SQLStatements.hpp]: Space here not necessary

**Review by Orange-hanter — CHANGES_REQUESTED** [review]

File: DataAccess/DataAccess.SQLite/LiteAdapter.cpp (hunk 1 above)
Comment (ivanboika) [reply DataAccess/DataAccess.SQLite/LiteAdapter.cpp]: done

File: DataAccess/DataAccess.SQLite/LiteAdapter.cpp (hunk 2 above)
Comment (ivanboika) [reply DataAccess/DataAccess.SQLite/LiteAdapter.cpp]: done

File: DataAccess/Public/Include/DataAccess/SQLStatements.hpp (hunk 3 above)
Comment (ivanboika) [reply DataAccess/Public/Include/DataAccess/SQLStatements.hpp]: Erased one empty line

**Review by Orange-hanter — APPROVED** [review]

**Pull request merged**